Admin/AdminInterface.php

```diff
@@ -937,6 +937,7 @@ public function getLabel();
     public function getPersistentParameters();
 
     /**
+     * NEXT MAJOR: remove this signature
```

Comment:
In the other side, with your method, the user will still have to implement a method that will be "brutally" removed on next major but will have not choice to keep it.